help/en/package/jmri/jmrit/ctc/CTC.shtml

```diff
@@ -5,236 +5,238 @@
   <title>JMRI: CTC</title><!--#include virtual="/help/en/parts/Style.shtml" -->
 </head>
 <body>
-  <!--#include virtual="/help/en/parts/Header.shtml" -->
+<!--#include virtual="/help/en/parts/Header.shtml" -->
 
-  <div id="mBody">
-    <div id="mainContent" class="no-sidebar">
-      <h1>JMRI: CTC</h1>
+<div id="mBody">
+  <div id="mainContent" class="no-sidebar">
+    <h1>JMRI: CTC</h1>
 
-      <p>CTC is a very complex system. An understanding of the goals of CTC and how it functions is
+    <p>Union Switch and Signal (US&S) CTC (Centralized Traffic Control) is a chiefly North American system for implementing a
```

Comment:
US&S should be `US&amp;S`

Reply:
@ralflang
This needs to be fixed before we can merge the PR.

Reply:
@dsand47
One option is that you approve the PR and that I fix this issue after this PR is merged. It's an easy fix.

Reply:
> US&S should be `US&amp;S`

Fixed by PR #13096.

Reply:
Sorry I missed that reply. We had a long weekend due to a public holiday.